src/llmq/dkgsession.cpp

```diff
@@ -989,14 +989,15 @@ void CDKGSession::SendCommitment(CDKGPendingMessages& pendingMessages)
     qc.sig = WITH_LOCK(activeMasternodeInfoCs, return activeMasternodeInfo.blsKeyOperator->Sign(commitmentHash));
     qc.quorumSig = skShare.Sign(commitmentHash);
 
+    const bool is_bls_legacy = bls::bls_legacy_scheme.load();
```

Comment:
please do this at the call site to avoid an atomic read when we basically never lie